src/mpid/ch4/netmod/ofi/ofi_init.h

```diff
@@ -14,6 +14,11 @@
 #include "ofi_impl.h"
 #include "mpir_cvars.h"
 #include "pmi.h"
+#ifdef USE_PMI2_API
```

Comment:
Does `USE_CRAYPMI_API` imply this macro as well? It is not set by your `subconfigure.m4` changes.